authors/apps/articles/models.py

```diff
@@ -7,7 +7,7 @@
 
 class Article(models.Model):
 
-    slug = models.SlugField(max_length=255)
```

Comment:
This change is not required because the slug is generated internally and not by a user so unless there is a reason that supersedes that, I request that you revert it.

Reply:
This change was made because I use the slug as a ForeignKey to the article model from the comment model. Without adding a unique constrainst, I would not be able to use the slug in this way.

Reply:
Let's make the change in the comments' serializer file to allow the article field refer to the slug rather than the id of articles.
You can look at this section of the DRF documentation on how to go about this -> `https://www.django-rest-framework.org/api-guide/relations/`

Reply:
This has been implemented. Thank you.